src/com/palmergames/bukkit/towny/object/Resident.java

```diff
@@ -110,16 +110,16 @@ public void setJailed(boolean isJailed) {
 			TownyUniverse.getInstance().getJailedResidentMap().remove(this);
 	}
 	
-	public void sendToJail(Player player, Integer index, Town town) {
+	public void sendToJail(Integer index, Town town) {
```

Comment:
Should probably use the primitive `int` parameter. Applies for the methods below as well.